eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -23,6 +21,7 @@ parameters:
   TestParallelization: '1'
   TestVersionSupport: $(TestVersionSupport)
   UseHttpFaultInjector: false
+  OSName:
```

Comment:
Does this need a default?

Reply:
No default, so that it's required to be passed in.